active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,622 @@
+but will not necessarily break user flow.
+
+An InfoBar will take up space in your layout and behave like any other child elements. 
+It will not cover up other content or float on top of it.
+
+Do not use an InfoBar control to confirm or respond directly to a user action, for time-sensitive alerts, 
+or for non-essential messages.
+
+## Remarks
+Use an InfoBar that is dismissed by the user or when the status is resolved for scenarios 
+that **directly** impact app perception or experience
+
+
+Here are some examples:
+- Internet connectivity lost
+- Error while saving a document when triggered automatically, not related to specific user action
+- No microphone plugged in when attempting to record
+- Can't connect to your phone
+- The subscription to the application is expired
+
+
+Use an InfoBar that is dismissed by the user for scenarios that **indirectly** impact app perception or experience
+
+Here are some examples:
+- A call has begun recording
+- Update applied with link to 'Release Notes'
+- The terms of service have been updated and require acknowledgement
+- An app-wide backup has successfully, asynchronously completed
+- The subscription to the application is close to expiring
+
+## When should a different control be used?
+
+There are some scenarios where a [ContentDialog](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.ContentDialog), 
+[Flyout](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.Flyout), or 
+[TeachingTip](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.TeachingTip) may be more appropriate to use.
+
+- For scenarios where a persistent notification is not needed, e.g. displaying information in context of 
+a specific UI element, a 
+[Flyout](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/flyouts) 
+is a better option. 
+- For scenarios where the application is confirming a user action, showing information the user ***must*** read, use a
+[ContentDialog](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/dialogs).
+  - Additionally, if a status change to the app is so severe that it needs to block all further ability for the user to interact with the app, use a ContentDialog.
+- For scenarios where a notification is a transient teaching moment, a 
+[TeachingTip](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/teaching-tip) 
+is a better option.
+
+
+For more info about choosing the right notification control, see the 
+[Dialogs and Flyouts](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/dialogs-and-flyouts/)
+article.
+
+
+# Examples
+
+## Create an InfoBar
+The XAML below describes an inline InfoBar with the default styling for an informational notification. 
+An info bar can be created anywhere in the element tree or code behind and will expand 
```

Comment:
"anywhere" in the tree -- does the parent need to support (multiple) children? E.g. could I put it on a `Border`, even if the `Border` already has content?

Reply:
There is nothing special about InfoBar in this sense. It is just another control. Border supports only one child, so this can be the child of the border. If you need multiple children, you need to add some panel (like StackPanel or Grid) and then add InfoBar to that panel.